test/cqlpy/test_tools.py

```diff
@@ -1731,3 +1731,25 @@ def test_scylla_sstable_query_temp_dir(cql, scylla_path, scylla_data_dir):
         assert res.returncode == 2
         assert res.stdout == ""
         assert res.stderr.endswith(f"error running operation: std::filesystem::__cxx11::filesystem_error (error generic:20, filesystem error: temp_directory_path: Not a directory [{f.name}])\n")
+
+
+@pytest.mark.parametrize("tag", [
+        "sstable-dump-4d1cc6a4-6c13-11f0-a3c9",
+        "test-4d1cc6a46c1311f0a3c9"
+])
+def test_scylla_sstable_dump_data_from_snapshot(cql, test_keyspace, scylla_path, scylla_data_dir, tag):
```

Comment:
It's impossible for a reader to guess what this test is testing and why, and what is special about the two tags you parametrized above. Please add a comment explaining what you are trying to reproduce here (please include a reference to the issue you're reproducing), and what kind of failure we saw in the past. In particular, in the code below, I see you just check that the "scylla sstables" command doesn't fail (you don't care what its actual output is) - so I assume it used to fail?

Reply:
> I see you just check that the "scylla sstables" command doesn't fail (you don't care what its actual output is) - so I assume it used to fail?

Yes, it is failing as of https://github.com/scylladb/scylladb/issues/25242 since the tool generates a wrong schema internally to query the sstable.

Will add json.loads validation on the tool's output to make that not only it succeeded, but it returned valid output